cloudflare/resource_cloudflare_healthcheck.go

```diff
@@ -225,14 +230,20 @@ func resourceCloudflareHealthcheckCreate(d *schema.ResourceData, meta interface{
 		return errors.Wrap(err, fmt.Sprintf("error creating healthcheck struct"))
 	}
 
-	hc, err := client.CreateHealthcheck(zoneID, healthcheck)
-	if err != nil {
-		return errors.Wrap(err, fmt.Sprintf("error creating standalone healthcheck"))
-	}
+	return resource.Retry(d.Timeout(schema.TimeoutCreate), func() *resource.RetryError {
+		hc, err := client.CreateHealthcheck(zoneID, healthcheck)
+		if err != nil {
+			if strings.Contains(err.Error(), "no such host") {
+				return resource.RetryableError(fmt.Errorf("hostname resolution failed"))
+			}
 
-	d.SetId(hc.ID)
+			return resource.NonRetryableError(errors.Wrap(err, fmt.Sprintf("error creating standalone healthcheck")))
+		}
 
-	return resourceCloudflareHealthcheckRead(d, meta)
+		d.SetId(hc.ID)
+
+		return resource.NonRetryableError(resourceCloudflareHealthcheckRead(d, meta))
```

Comment:
Did you intend to bring in #789? Or could we pick this one out of this PR?

Reply:
Heh, speedy recovery 😛

Reply:
Local git weirdness. Should be fixed with the latest merge